pyproject.toml

```diff
@@ -28,6 +28,7 @@ packages = [
 [tool.poetry.dependencies]
 bs4 = "^0.0.1"
 circuit-maintenance-parser = "^2.0.0"
+exchangelib = "^5.1.0"
```

Comment:
as I saw it, I guess it would make sense to repack the optional packages to  let the user which kind of source will use. For example, they don't need to install exchangelib if are using google-auth and vice-versa.
But, I would leave this for another PR.